# Sequencing, fast and slow: profiling microbiomes in human samples with nanopore sequencing

## Information on Files
`qc_wimp_stats.py` is used to extract necessary information from the QC and WIMP files, and `interaction_classification.py` file is used to classify the reads into different host-microbiome interaction types and get summary statistics for different samples. Then, `plot_interaction.py` is used to plot these interaction types to a stacked bar graph.

`find_unclassified.ipynb` and `find_unclassified_combined.ipynb` files are used to further identify the unclassified microorganisms.


## Modules & Libraries
- Python 3.8.3 (https://www.python.org)
- pandas v.1.0.5 (https://github.com/pandas-dev/pandas)


# Contributors
- [Yunseol Park](https://github.com/YunseolPark)
- [Jeesu Lee](https://github.com/jeesu-97)
- [Hyunjin Shim](https://github.com/hjshim)
